inc/template-tags.php

```diff
@@ -325,6 +320,39 @@ function _s_display_social_icons() {
 }
 
 /**
+ * Get the Twitter social sharing URL for the current page.
```

Comment:
@kellenmace Honest question, Do you think these getters should be moved to `extras.php` or maybe `queries.php`?

I'd like to keep `template-tags.php` as lean as possible...